packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -64,6 +67,60 @@ enum NavigationDecision {
   navigate,
 }
 
+/// Android [WebViewPlatform] that uses [AndroidViewSurface] to build the [WebView] widget.
+///
+/// To use this, set [WebView.platform] to an instance of this class.
```

Comment:
Maybe add some guidance on when should one use it and what are the costs?

Reply:
done. I added it below. @blasten What are your thoughts on the explanation below?